backends-common/elasticsearch-v7/pom.xml

```diff
@@ -87,12 +87,16 @@
         <dependency>
             <groupId>org.elasticsearch.client</groupId>
             <artifactId>elasticsearch-rest-high-level-client</artifactId>
-            <version>7.10.2</version>
+            <version>7.17.3</version>
```

Comment:
Not part of your commit description, and unnecessary as we try to upgrade ES to 8 and that dependency disappears, replaced by an other client

Reply:
yes, my miss, I see your pr.